mmv1/third_party/terraform/services/composer/resource_composer_environment.go.erb

```diff
@@ -682,6 +687,35 @@ func ResourceComposerEnvironment() *schema.Resource {
 								},
 							},
 						},
+
+						"data_retention_config": {
+							Type:         schema.TypeList,
+							Optional:     true,
+							Computed:     true,
+							AtLeastOneOf: composerConfigKeys,
+							MaxItems:     1,
+							Description:  `The configuration setting for Airflow data retention mechanism. This field is supported for Cloud Composer environments in versions composer-2.0.32-airflow-2.1.4. or newer`,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"task_logs_retention_config": {
+										Type:         schema.TypeList,
+										Optional:     true,
+										Description: `Optional. The configuration setting for Task Logs.`,
+										AtLeastOneOf: dataRetentionConfigKeys,
```

Comment:
Just want to double-check that this is needed here, since task_logs_retention_config is the only child of data_retention_config? Essentially this is saying that the field is required (until more children are added). I'd recommend making this field `Required: true` instead; making it optional + atleastoneof later would not be a breaking change. (Unless you're planning to do an immediate follow-on PR like next week to add more sibling fields.)